src/Ilios/ApiBundle/Resources/swagger/definitions/CalendarEvent.yml

```diff
@@ -0,0 +1,44 @@
+CalendarEvent:
```

Comment:
I don't think we can do the entire model as readOnly.  I marked the properties and in the docs it only shows up in GET requests.